api/services/CalendarEvent/index.js

```diff
@@ -8,9 +8,24 @@ import slugify from 'slugify';
 
 export default class CalendarEvent extends BaseModelService {
   @saveInstance
-  create(body) {
+  @withTransaction
```

Comment:
What is the purpose of the transaction when you are not passing it to the create query.